src/pages/Portfolio/ImpactDashboard/LendingInsights.vue

```diff
@@ -1,8 +1,8 @@
 <template>
 	<async-portfolio-section @visible="fetchAsyncData" data-testid="lending-insights" class="!tw-bg-eco-green-4">
-		<h2 class="tw-mb-3 md:tw-mb-2 tw-text-white">
+		<h3 class="tw-mb-3 md:tw-mb-2 tw-text-white tw-text-center md:tw-text-left">
```

Comment:
Instead of changing the semantic tag (because this is still structurally an h2) use the h3 styling class `tw-text-h3`